docs/source/markdown/podman-images.1.md

```diff
@@ -73,16 +73,33 @@ Change the default output format.  This can be of a supported type like 'json'
 
-| **Placeholder** | **Description**                                                               |
-| --------------- | ----------------------------------------------------------------------------- |
-| .ID             | Image ID                                                                      |
-| .Repository     | Image repository                                                              |
-| .Tag            | Image tag                                                                     |
-| .Digest         | Image digest                                                                  |
-| .CreatedSince   | Elapsed time since the image was created			     					  |
-| .CreatedAt      | Time when the image was created                                               |
-| .Size           | Size of layer on disk                                                         |
-| .History        | History of the image layer                                                    |
+| **Placeholder**   | **Description**                                    |
+|-------------------|----------------------------------------------------|
+| .Containers       | Number of containers using this image              |
+| .Created          | Elapset time since the image was created           |
+| .CreatedAt        | Time when the image was created, YYYY-MM-DD-etc    |
+| .CreatedSince     | Same as .Created                                   |
+| .CreatedTime      | Same as .CreatedAt                                 |
+| .Dangling         | Same as .IsDangling                                |
+| .Digest           | Image digest                                       |
+| .History          | History of the image layer                         |
+| .ID               | Image ID (truncated)                               |
+| .Id               | Image ID (full SHA)                                |
+| .IsDangling       | Is image dangling? (true/false)                    |
+| .IsReadOnly       | Is unage read-only? (true/false)                   |
+| .Labels           | map[] of labels                                    |
+| .Names            | Image FQIN                                         |
+| .ParentId         | Full SHA of parent image ID, or null (string)      |
+| .ReadOnly         | Same as .IsReadOnly                                |
+| .RepoDigests      | map[] of zero or more repo/name@sha256:SHA strings |
+| .Repository       | Image repository                                   |
+| .RepoTags         | map[] of zero or more FQIN strings for this image  |
+| .SharedSize       | Always seems to be 0                               |
+| .Size             | Size of layer on disk (human-friendly string)      |
+| .Tag              | Image tag                                          |
+| .VirtualSize      | Size of layer on disk (bytes)                      |
+
+[//]: # (.ImageSummary - INTERNAL. This silences xref-helpmsgs-manpages)
```

Comment:
Thanks, @TomSweeneyRedHat . Nice catch on the typo; on the `-etc`, I think I should just include the explicit HHMM. I will apply the fixes on a subsequent push... but for now, a reminder that this line right here is the crux of the PR. This line right here hides `.ImageSummary` from the man page. My question is:
* should we *document* weird fields like `.ImageSummary`, or *hide* them?

The more I think about it, the more I hate the *hide* option. As I see it, the man page is a crucial resource for people using podman in ways we can't foresee. Few people are going to use every possible option, but they will want an overview of what's available. And someone playing with the command completion will experiment, and turn to the man page for help. They will, IMO, prefer to see an acknowledgment like "this option exists, but it's not for people like you" over "we will not even acknowledge that this option exists".

But it's not my decision to make. Again, we cannot hide these options from command-line completion, so people *will* find them. The question is, what then?